.github/workflows/gsy-e_ci.yaml

```diff
@@ -21,9 +21,16 @@ jobs:
       env:
         prBody: ${{ steps.PR.outputs.pr_body }}
       run: |
-        echo "::set-output name=INTEGRATION_TESTS_BRANCH::$(echo -e $prBody | sed -n 's/.*\*\*INTEGRATION_TESTS_BRANCH\*\*=\([^ ]*\).*/\1/p')"
+        echo "::set-output name=PARSED_BRANCH::$(echo -e $prBody | sed -n 's/.*\*\*INTEGRATION_TESTS_BRANCH\*\*=\([^ ]*\).*/\1/p')"
       id: parse_branch
 
+    - name: validate parsed branch
+      env:
+        PARSED_BRANCH: ${{ steps.parse_branch.outputs.PARSED_BRANCH }}
+      run: |
+        echo "::set-output name=INTEGRATION_TESTS_BRANCH::$( [ -z $PARSED_BRANCH ] && echo -e 'master' || echo -e $PARSED_BRANCH)"
```

Comment:
Could you please explain why `echo` is needed in this part?
```sh
$( [ -z $PARSED_BRANCH ] && echo -e 'master' || echo -e $PARSED_BRANCH)
```

Reply:
In any case, I think this syntax might be more suitable:
```sh
${PARSED_BRANCH:-'master'}
```
(Based on bash's [Shell Parameter Expansion](https://www.gnu.org/software/bash/manual/bash.html#Shell-Parameter-Expansion))